src/systems/userinput/userinput-debug.js

```diff
@@ -3,11 +3,44 @@ AFRAME.registerSystem("userinput-debug", {
   active: true,
 
   tick() {
-    if (!this.active) {
-      return;
-    }
+    if (!this.active) return;
+
     const userinput = AFRAME.scenes[0].systems.userinput;
+
+    if (this.userinputFrameStatus) {
+      const replacer = (k, v) => {
+        if (typeof v === "number") {
+          return `${v >= 0 ? "+" : ""}${v.toFixed(3)}`;
+        }
+        return v;
+      };
+      const pathsOutput = [];
+      const { frame } = userinput;
+      for (const key in frame) {
+        if (!frame.hasOwnProperty(key)) continue;
+        if (!["/wmr-user", "/actions/"].some(x => key.startsWith(x))) continue;
```

Comment:
Did you mean to leave `/wmr-user` in here? Seems like it should only be actions by default.